test/functional/wallet_multiwallet.py

```diff
@@ -125,7 +125,8 @@ def wallet_file(name):
         self.start_node(0, ['-wallet=w4', '-wallet=w5'])
         assert_equal(set(node.listwallets()), {"w4", "w5"})
         w5 = wallet("w5")
-        w5.generate(1)
+        self.nodes[0].generatetoaddress(nblocks=1, address=w5.getnewaddress())
```

Comment:
Just noting that could use `w5.generatetoaddress` despite the fact this RPC doesn't require a wallet.

Reply:
Yes, that's possible, but I think it's better to call the `generatetoaddress` method on the node endpoint rather than the wallet endoint (since it's a node method).

However, I have updated these calls to use the `node` alias rather than `self.nodes[0]` to be mode compact.